### PR TITLE
Add additional ML cluster settings

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
Signed-off-by: Naarcha-AWS <naarcha@amazon.com>

### Description

These settings were added to 2.4 post-release via patch. Will backport to both 2.5 and 2.4.

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by ylwu-amzn — APPROVED** [review]

**Review by ariamarble — APPROVED** [review]
lgtm

File: _ml-commons-plugin/cluster-settings.md [hunk 1]
```diff
@@ -62,7 +91,7 @@ plugins.ml_commons.sync_up_job_interval_in_seconds: 10
 
 ## Predict monitoring requests
 
-Controls how many upload model tasks can run in parallel on one node. If set to `0`, you cannot upload models to any node.
+Controls how many predict requests are monitored on one node. If set to `0`, OpenSearch clears all monitoring predict request in cache and does monitor for new predict requests.
```
Comment (ylwu-amzn) [inline _ml-commons-plugin/cluster-settings.md:94]: does *not monitor for new predict requests?

File: _ml-commons-plugin/cluster-settings.md [hunk 2]
```diff
@@ -62,7 +91,7 @@ plugins.ml_commons.sync_up_job_interval_in_seconds: 10
 
 ## Predict monitoring requests
 
-Controls how many upload model tasks can run in parallel on one node. If set to `0`, you cannot upload models to any node.
+Controls how many predict requests are monitored on one node. If set to `0`, OpenSearch clears all monitoring predict request in cache and does not monitor for new predict requests.
```
Comment (cwillum) [inline _ml-commons-plugin/cluster-settings.md:94]: "requests", right?
"If set to `0`, OpenSearch clears all monitoring predict requests in cache ..."

File: _ml-commons-plugin/cluster-settings.md [hunk 3]
```diff
@@ -73,7 +102,7 @@ plugins.ml_commons.monitoring_request_count: 100
 ### Value range
 
 - Default value: `100`
-- Value range: [0, 100,000,000]
+- Value range: [0, 10,000,000]
```
Comment (cwillum) [inline _ml-commons-plugin/cluster-settings.md:105]: Does the system interpret commas in numbers values? In other words, if you added this as a value, the system could read the number 10,000,000? 
If yes, you might add a comma to 10,000 in line 73.

**Review by cwillum — APPROVED** [review]
LGTM. A couple comments.

File: _ml-commons-plugin/cluster-settings.md (hunk 2 above)
Comment (Naarcha-AWS) [reply _ml-commons-plugin/cluster-settings.md]: Fixed

File: _ml-commons-plugin/cluster-settings.md (hunk 3 above)
Comment (Naarcha-AWS) [reply _ml-commons-plugin/cluster-settings.md:105]: Fixed the three instances where we didn't use commas previously.

**Pull request merged**